src/kosha/db/kriti.clj

```diff
@@ -0,0 +1,21 @@
+(ns kosha.db.kriti
+  (:require [kosha.db.util :as db-util]))
+
+(defn kriti
+  "Retrieve details of a kriti given it's kriti_id."
+  [kriti-id]
+  (let [q ["SELECT k.kriti_id, k.name, k.composer, k.taala, k.url, k.lyrics, k.meaning, k.ragam_id, k.data_source, r.name AS ragam_name
```

Comment:
If you need a list of fields that come out of the DB, you could make that a list in clojure, and do a `select-keys` in the result, that'd be more maintainable, and generally applicable.

Slightly tangential to this, it'd be good to have a automatically generated schema dump checked into the repo for reference purposes.